libs/expo/betterangels/src/lib/screens/AddEditClient/index.tsx

```diff
@@ -391,7 +393,7 @@ export default function AddEditClient({ id }: { id?: string }) {
             )}
           </MainScrollContainer>
           <BottomActions
-            disabled={!!methods.formState.errors.californiaId}
+            disabled={!!methods.formState.errors.californiaId || isCreating || isUpdating}
```

Comment:
it looks like you only have to add `isCreating` and `isUpdating` to `isLoading`, so you can remove it here. Updating either prop will have the same effect, but `isLoading` is more appropriate, since we're specifically passing a loading state

`libs/expo/shared/ui-components/src/lib/BottomActions/index.tsx` ([link](https://github.com/BetterAngelsLA/monorepo/blob/main/libs/expo/shared/ui-components/src/lib/BottomActions/index.tsx#L11-L12)) is a bit confusing, because it takes both a `disabled` prop and an `isLoading` prop. But the `<Button>` component's `disabled` prop combines them:
```js
<Button
  loading={isLoading}
  disabled={isLoading || disabled}
  fontSize="sm"
  height="md"
  variant="primary"
  style={{ minWidth: 90 }}
  accessibilityHint="submit the form"
  title={submitTitle}
  onPress={onSubmit}
/>
```
which is why passing the loading state to either one will have the same effect